### ISSUE TITLE
Subtitles

### ISSUE THREAD

**Issue by pepijn809** [senpaiski/TopAnimeStream#21]
The Subtitles "English: Signs and Songs" don't really seem to work. Not sure if it's meant for japanese people who only want to see signs translated. But if it is for full translation with signs to english, than there is probably something wrong. 

I noticed this on the Anime "The Devil is a Partimer". I think it may not be a bug, but still worth taking a look at.

**Comment by senpaiski** [comment]
Kk I'll check this one tonight. 

I don't know if you have notice but subtitles are out of sync by a fraction of milliseconds! xD Quick fix! Go to video-tas.js and change 3.18 to 3.8 instead!

https://github.com/senpaiski/TopAnimeStream/blob/dev/public/js/video-tas.js

**Comment by pepijn809** [comment]
Yeah, i did notice some pieces where it skipped funny. So i guess that is related to that same issue ;)
Will update my version :)

**Comment by senpaiski** [comment]
Ah makes sense now! :P
